app/models/Amounts.scala

```diff
@@ -0,0 +1,15 @@
+package models
+
+case class Amount(value: String, isDefault: Option[Boolean])
+
+case class Amounts(ONE_OFF: List[Amount], MONTHLY: List[Amount], ANNUAL: List[Amount])
```

Comment:
so UI-wise, this is independent of the contribution types as set in the JSON file? if you've only got "one-off" enabled for the US, say, you'll still be able to set amounts for all three types

Reply:
Correct, contributionTypes and amounts are still configured separately. This is a bit confusing and perhaps the 2 config files can be unified later